Updates

singh-manavv opened the pull request:
It adds the feature to enter the amount to craft.
It will show craftable items at the top of list and disabled ones after that.

BerkieBb commented:
There is no table called Config.CraftingItems['items'], this is only the case for attachmentcrafting as it has a location set.

singh-manavv committed:
Updates

It adds the feature to enter the amount to craft.
It will show craftable items at the top of list and disabled ones after that

singh-manavv committed:
conflicts resolve

singh-manavv committed:
resolve conflicts
